packages/compiler/src/init/init-template.ts

```diff
@@ -49,6 +53,12 @@ export interface InitTemplate {
    * List of files to copy.
    */
   files?: InitTemplateFile[];
+
+  /**  */
+  welcomeMessage?: string;
```

Comment:
could we group maybe all messages together
```
messages: {
  welcome: string
  post: string
}
```

Reply:
or should we have something similar to dotnet template with post actions

```
postActions: [{type: "message", content: string}}
```

Reply:
I like the postActions. But we want it better designed than just have message action. Let's open a backlog issue to track the design, potentially reference the design of [dotnet template](https://github.com/dotnet/templating/wiki/Post-Action-Registry). 

I will remove the messages for now. The Welcome message will just be the description then.

Reply:
#2125 created.

Reply:
yeah thats a good idea